src/materials/P91LAROMANCEStressUpdate.C

```diff
@@ -0,0 +1,5324 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/*                       BlackBear                              */
+/*                                                              */
+/*           (c) 2017 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "P91LAROMANCEStressUpdate.h"
+
+registerMooseObject("BlackBearApp", P91LAROMANCEStressUpdate);
+
+InputParameters
+P91LAROMANCEStressUpdate::validParams()
+{
+  InputParameters params = ADLAROMANCEStressUpdateBase::validParams();
+  params.addClassDescription("LAROMANCE creep update model for P91");
+
+  // Override defaults for material specific parameters below
+  params.addRangeCheckedParam<Real>("initial_cell_dislocation_density",
+                                    5.0e12,
+                                    "initial_cell_dislocation_density >=1.0+12 & "
+                                    "initial_cell_dislocation_density <= 6.0E+12",
+                                    "Initial density of cell (glissile) dislocations (1/m^2)");
+  params.addRangeCheckedParam<Real>(
+      "max_relative_cell_dislocation_increment",
+      0.5,
+      "max_relative_cell_dislocation_increment > 0.0",
+      "Maximum increment of density of cell (glissile) dislocations.");
+  params.addRangeCheckedParam<Real>(
+      "initial_wall_dislocation_density",
+      6.0e12,
+      "initial_wall_dislocation_density >= 6.0e12 & "
+      "initial_wall_dislocation_density <= 6.0e12",
```

Comment:
For consistency, I've modified the above lines with the following, based on Aaron's email:
```
  params.addRangeCheckedParam<Real>("initial_cell_dislocation_density",
                                    4.0e12,
                                    "initial_cell_dislocation_density >=1.0+12 & "
                                    "initial_cell_dislocation_density <= 6.0E+12",
                                    "Initial density of cell (glissile) dislocations (1/m^2)");
  params.addRangeCheckedParam<Real>(
      "max_relative_cell_dislocation_increment",
      0.5,
      "max_relative_cell_dislocation_increment > 0.0",
      "Maximum increment of density of cell (glissile) dislocations.");
  params.addRangeCheckedParam<Real>(
      "initial_wall_dislocation_density",
      10.0e12,
      "initial_wall_dislocation_density >= 6.0e12 & "
      "initial_wall_dislocation_density <= 18.0e12",
      "Cell wall (locked) dislocation density initial value (1/m^2).");
```